sensor_msgs/CompressedImage img1
sensor_msgs/CompressedImage img2
sensor_msgs/CompressedImage img3
sensor_msgs/CompressedImage img4
